docs/sequence-diagrams/generate-uml.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+java -jar plantuml.jar sidecar.puml
```

Comment:
mhm is there some sequential diagram language that does not require java? This is blocker for CI I assume.